lang/rust/Makefile

```diff
@@ -3,16 +3,15 @@
 # Copyright (C) 2023 Luca Barbato and Donald Hoskins
 
 include $(TOPDIR)/rules.mk
-include ./rust-values.mk
 
 PKG_NAME:=rust
 PKG_VERSION:=1.72.0
-PKG_RELEASE:=1
+PKG_RELEASE:=2
 
 PKG_SOURCE:=rustc-$(PKG_VERSION)-src.tar.gz
 PKG_SOURCE_URL:=https://static.rust-lang.org/dist/
 PKG_HASH:=ea9d61bbb51d76b6ea681156f69f0e0596b59722f04414b01c6e100b4b5be3a1
-HOST_BUILD_DIR:=$(BUILD_DIR)/host/rust-$(RUSTC_TARGET_ARCH)/rustc-$(PKG_VERSION)-src
+HOST_BUILD_DIR:=$(BUILD_DIR)/host/rustc-$(PKG_VERSION)-src
```

Comment:
this seems unrelated

Reply:
I think it is better to include rust-values.mk after package.mk for the Rust package itself, in case code in rust-values.mk needs to access `PKG_*` variables to build rustc in the future. To do this it is necessary to remove the reference to `$(RUSTC_TARGET_ARCH)` here, because this variable would not be defined yet.

Reply:
You need to have that otherwise you break some people use-case :(

Reply:
When @1715173329 changed the build dir from `$(BUILD_DIR_HOST)` to `$(BUILD_DIR)/host` in efdbac3, this had the same effect as including `$(RUSTC_TARGET_ARCH)` in the path. `$(BUILD_DIR)` is different for each target.

Reply:
Then it seems good :) Thank you for doublechecking.